# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1690653315 -7200
# Parent  63dd1a82f45c39ae1f0e0034badf51baa767a07d
Misc: Reduce excessive padding of unified toolbar.

diff --git a/mail/themes/shared/mail/unifiedToolbar.css b/mail/themes/shared/mail/unifiedToolbar.css
--- a/mail/themes/shared/mail/unifiedToolbar.css
+++ b/mail/themes/shared/mail/unifiedToolbar.css
@@ -1,27 +1,27 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 @import url("chrome://messenger/skin/shared/unifiedToolbarShared.css");
 
 :root {
-  --unified-toolbar-padding: 7px;
+  --unified-toolbar-padding: 2px;
   --unified-drag-space: 30px;
   --unified-customization-padding: 30px;
 }
 
 :root[uidensity="compact"] {
-  --unified-toolbar-padding: 4px;
+  --unified-toolbar-padding: 0;
   --unified-drag-space: 15px;
 }
 
 :root[uidensity="touch"] {
-  --unified-toolbar-padding: 10px;
+  --unified-toolbar-padding: 4px;
 }
 
 :root[sizemode="fullscreen"],
 :root:not([tabsintitlebar]) {
   --unified-drag-space: 0;
 }
 
 unified-toolbar {
